src/kw/kw.cpp

```diff
@@ -57,9 +60,7 @@ bool KanoWorld::login(const string& username, const string& password,
     std::shared_ptr<JSON_Value> res;
 
     try {
-        res = this->http_client->POST(
-            "https://worldapi-dev.kano.me/accounts/auth",
-            body);
+        res = this->http_client->POST(this->api_url + "/accounts/auth", body);
```

Comment:
Perhaps have some tests with a UTF-8 password here?

Reply:
Would love to but without creating accounts on the server I'm not sure how to do this. Need to investigate.

Reply:
Maybe for now we can double check that `Poco` doesn't freak out in this case?